client/lib/domains/get-domain-type-text.js

```diff
@@ -15,11 +15,20 @@ function translatePlaceholder( string ) {
  *
  * @param   {Object}   domain Domain object
  * @param   {Function} __     Translate function
+ * @param   {string} context  Context of the returned text (DOMAIN_ITEM: item of a domain list, PAGE_TITLE: title when managing the domain)
  * @returns {string}          Domain type text
  */
-export function getDomainTypeText( domain = {}, __ = translatePlaceholder ) {
+export function getDomainTypeText(
+	domain = {},
+	__ = translatePlaceholder,
+	context = domainTitleContext.DOMAIN_ITEM
+) {
 	switch ( domain.type ) {
 		case domainTypes.MAPPED:
+			if ( context === domainTitleContext.PAGE_TITLE ) {
+				return __( 'Connected domain' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Domain Connected' )` *ES Score: 8*